spark/src/test/scala/org/apache/spark/sql/delta/rowid/RowIdTestUtils.scala

```diff
@@ -234,5 +234,4 @@ trait RowIdTestUtils extends RowTrackingTestUtils with DeltaSQLCommandTest with
   def checkRowTrackingMarkedAsPreservedForCommit(log: DeltaLog)(operation: => Unit): Unit = {
     assert(rowTrackingMarkedAsPreservedForCommit(log)(operation))
   }
-
 }
```

Comment:
nit: undo this spurious change

Reply:
@johanl-db Why not? I think it's worth doing as I added that by mistake in the Preserve Delete PR.

Reply:
Oh, alright then